### PR TITLE
Add example pc call to documentation usage section

### PR DESCRIPTION
**Pull request by ChristopherSchmidt89**
also link to public API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,28 @@ python3 -m gpucsl --gaussian -d ./data.csv -o . -l 3
...
+The following code snippet provides a small example for calling the `pc` function:
+```python
+import numpy as np
+from gpucsl.pc.pc import pc, DataDistribution
+from gpucsl.pc.helpers import correlation_matrix_of
+
+samples = np.random.rand(1000,10)
+max_level = 3
+alpha = 0.05
+correlation_matrix = correlation_matrix_of(samples)
```
Comment (BraunTom) [inline README.md]: You are not forced to calculate the correlation matrix yourself . You can simply leave it and gpucsl calculates it for you. Or do you want to show something by probiding it?

Comment (ChristopherSchmidt89) [reply README.md]: Good point, I removed it 👍

File: README.md [hunk 2]
```diff
@@ -50,7 +50,25 @@ python3 -m gpucsl --gaussian -d ./data.csv -o . -l 3
...
+
+The following code snippet provides a small example for calling the `pc` function:
+```python
+import numpy as np
+from gpucsl.pc.pc import pc, DataDistribution
+
+samples = np.random.rand(1000,10)
+max_level = 3
+alpha = 0.05
+res = pc(
```
Comment (BraunTom) [inline README.md]: we probably want to provide it at least with `(pc_result, pc_runtime) = pc(...` as pc returns a tuple and pc_result contains the CPDAG (pc_result is a tuple too). The CPDAG is probably what people are interested in
(so probably even provide `((directed_graph, separation_sets, pmax, discover_skeleton_runtime, edge_orientation_runtime, discover_skeleton_kernel_runtime), pc_runtime) = pc(...` or is that too much detail?)

Comment (ChristopherSchmidt89) [reply README.md]: Valid point, I think the descriptive return types make the example easier to understand

**Review by BraunTom — APPROVED** [review]

**Pull request merged**